### PR TITLE
doc: 15.2.14 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
My editor automatically fixed some formatting, hope it's OK

**Comment by tchaikov** [comment]
jenkins test make check arm64

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -5,6 +5,115 @@ Octopus
...
+* ceph: ignore BrokenPipeError when printing help (`pr#41586 <https://github.com/ceph/ceph/pull/41586>`_, Ernesto Puerta)
+* cephadm: fix escaping/quoting of stderr-prefix arg for ceph daemons (`pr#40948 <https://github.com/ceph/ceph/pull/40948>`_, Michael Fritch, Sage Weil)
+* cephadm: fix port_in_use when IPv6 is disabled (`pr#41602 <https://github.com/ceph/ceph/pull/41602>`_, Patrick Seidensal)
+* cephfs: client: add ability to lookup snapped inodes by inode number (`pr#40768 <https://github.com/ceph/ceph/pull/40768>`_, Jeff Layton, Xiubo Li)
+* cls/rgw: look for plain entries in non-ascii plain namespace too (`pr#41775 <https://github.com/ceph/ceph/pull/41775>`_, Mykola Golub)
+* cmake: build static libs if they are internal ones (`pr#39904 <https://github.com/ceph/ceph/pull/39904>`_, Kefu Chai)
+* crush/crush: ensure alignof(crush_work_bucket) is 1 (`pr#41622 <https://github.com/ceph/ceph/pull/41622>`_, Kefu Chai)
+* debian/control: ceph-mgr-modules-core does not Recommend ceph-mgr-rook (`pr#41878 <https://github.com/ceph/ceph/pull/41878>`_, Kefu Chai)
```
Comment (idryomov) [inline doc/releases/octopus.rst:37]: Do we need to highlight this one in Notable Changes, same as in 16.2.5?

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -1194,7 +1303,7 @@ Major Changes from Nautilus
 
 General
 ~~~~~~~
-  
+
```
Comment (idryomov) [inline doc/releases/octopus.rst:1306]: Nit: I would prefer no unrelated whitespace changes or, if it is really irritating and makes your editor go crazy, as a separate commit.

File: doc/releases/octopus.rst [hunk 3]
```diff
@@ -5,6 +5,115 @@ Octopus
...
+users update to this release.
+
+Notable Changes
+---------------
+
+* RGW: It is possible to specify ssl options and ciphers for beast frontend now.
+  The default ssl options setting is "no_sslv2:no_sslv3:no_tlsv1:no_tlsv1_1".
+  If you want to return back the old behavior add 'ssl_options=' (empty) to
+  ``rgw frontends`` configuration.
+
```
Comment (batrick) [inline doc/releases/octopus.rst:21]:
```suggestion
 * CephFS: old clusters (pre-Jewel) that did not use CephFS have legacy data structures in the ceph-mon stores. These structures are not understood by Pacific monitors. With Octopus v15.2.14, the monitors have been taught to flush and trim these old structures out in preparation for an upgrade to Pacific or Quincy. For more information, see `issue 51673 <https://tracker.ceph.com/issues/51673>`.
```

Also we should broadcast this fix to ceph-users/dev@ceph.io. Clusters created pre-Jewel that have never used CephFS should make a pit-stop upgrade to Octopus v15.2.14+ before moving on to Pacific.

**Review by batrick — CHANGES_REQUESTED** [review]

File: doc/releases/octopus.rst (hunk 1 above)
Comment (neha-ojha) [reply doc/releases/octopus.rst:37]: I will take care of this.

File: doc/releases/octopus.rst (hunk 3 above)
Comment (yuriw) [reply doc/releases/octopus.rst:21]: @batrick can you push it ?

Comment (batrick) [reply doc/releases/octopus.rst:21]: done

**Review by batrick — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]

**Comment by djgalloway** [comment]
Sorry, I didn't mean to sign all the commits.  Just Yuri's.  I was changing the release date to today.
